C#/C#.sublime-syntax

```diff
@@ -342,19 +342,19 @@ contexts:
       captures:
         1: keyword.declaration.class.cs
         2: entity.name.class.cs
-      push: [class_signature, data_type_signature]
+      push: [class_signature, data_type_constraint, maybe_constructor_parameters, data_type_signature]
```

Comment:
With this common schema, should `data_type_signature` be renamed to `data_type_generic` with only consuming `<...>` generics? It seems rather unlikely, that constraints appear in front of primary constructor parameters.

Something like that, I have in mind.

```yml
  data_type_generic:
    - match: '<'
      scope: meta.generic.cs punctuation.definition.generic.begin.cs
      push: data_type_generic_body
    - match: (?=\S)
      pop: true

  data_type_generic_body:
    - meta_content_scope: meta.generic.cs
    - include: type_parameter
```

Reply:
see also: https://github.com/forkeith/Packages/pull/30

Reply:
Thanks, merged